pkg/share/manager/sql/sql.go

```diff
@@ -216,7 +216,7 @@ func (m *mgr) Unshare(ctx context.Context, ref *collaboration.ShareReference) er
 			return err
 		}
 		owner := formatUserID(key.Owner)
-		query = "DELETE FROM oc_share WHERE uid_owner=? AND item_source=? AND share_type=? AND share_with=? AND (uid_owner=? or uid_initiator=?)"
+		query = "DELETE FROM oc_share WHERE uid_owner=? AND file_source=? AND share_type=? AND share_with=? AND (uid_owner=? or uid_initiator=?)"
```

Comment:
isn't this a breaking change? 🤔

Reply:
Yeah. Good catch. @butonic Was this change intended?